lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -80,7 +80,6 @@ export class RequestValidator {
         // Remove any query parameters already included in SSO params
         queryParams.forEach((value, key) => {
             if (eQParams[key]) {
-                console.log("Removed param " + key + " from extraQueryParameters since it was already present in library query parameters.");
```

Comment:
Didn't seem appropriate for this to throw an error, since it is just noting that extra parameters have been removed, and must continue to go through the rest of the params to remove duplicates. Not able to use a logger in static. Thinking that this log may not be necessary after all.

Reply:
I agree that error isn't appropriate. The log message is needed though, lets figure out how to fix this. Can be in a later PR.